ui/store/actions.ts

```diff
@@ -2298,44 +2298,6 @@ export function addTokens(
   };
 }
 
-export function rejectWatchAsset(
-  suggestedAssetID: string,
-): ThunkAction<void, MetaMaskReduxState, unknown, AnyAction> {
-  return async (dispatch: MetaMaskReduxDispatch) => {
-    dispatch(showLoadingIndication());
-    try {
-      await submitRequestToBackground('rejectWatchAsset', [suggestedAssetID]);
-      await forceUpdateMetamaskState(dispatch);
-    } catch (error) {
-      logErrorWithMessage(error);
-      dispatch(displayWarning(error));
-      return;
-    } finally {
-      dispatch(hideLoadingIndication());
-    }
-    dispatch(closeCurrentNotificationWindow());
-  };
-}
-
-export function acceptWatchAsset(
-  suggestedAssetID: string,
-): ThunkAction<void, MetaMaskReduxState, unknown, AnyAction> {
-  return async (dispatch: MetaMaskReduxDispatch) => {
-    dispatch(showLoadingIndication());
```

Comment:
For slower connections, do we lose the loading indicator now?

If so, maybe it's worth looking at a separate ticket to add this into the `resolvePendingApproval` and `rejectPendingApproval` actions.

Reply:
Up until now, resolving or rejecting an approval would happen very fast.

Now that we have the `expectsResult` option though, we should probably do what you suggest and add the loading indication for resolving actions.